v3/fleet-local/control/flight-director@.service

```diff
@@ -19,31 +19,31 @@ ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   --name flight-director \
   --net='host' \
   -e LOG_APP_NAME=flight-director \
-  -e FD_API_SERVER_PORT=`etcdctl get /FD/FD_API_SERVER_PORT` \
-  -e FD_CHRONOS_MASTER=`etcdctl get /FD/FD_CHRONOS_MASTER` \
-  -e FD_DB_DATABASE=`etcdctl get /FD/FD_DB_DATABASE` \
-  -e FD_DB_ENGINE=`etcdctl get /FD/FD_DB_ENGINE` \
+  -e FD_API_SERVER_PORT=`etcdctl get /flight-director/api-server-port` \
```

Comment:
I would move all of these into `/flight-director/config`. The `/flight-director` namespace is actually created & used by the app.